assertions/manager.go

```diff
@@ -106,6 +107,7 @@ func NewManager(
 	postInterval time.Duration,
 	averageTimeForBlockCreation time.Duration,
 	apiDB db.Database,
```

Comment:
Let's use an Opt for this `WithFastConfirmations()` instead of adding a bool

Reply:
Done.